h5py/h5p.pyx

```diff
@@ -1078,7 +1081,6 @@ cdef class PropFAID(PropInstanceID):
         - h5fd.MULTI
         - h5fd.SEC2
         - h5fd.STDIO
-        - h5fd.SPLIT
```

Comment:
I think we want to keep this line?

Reply:
This mode doesn't exist.  Split is implemented using the MULTI mode.

Reply:
At one point I introduced the mode, but then I realized it doesn't really exist so I removed the comment.